src/pages/models/[id].tsx

```diff
@@ -42,12 +43,12 @@ const dummyImages = [
     },
 ];
 
-export default function Page({ modelId, modelData }: Props) {
+export default function Page({ modelId, modelData, userIds }: Props) {
     const { webApi, editMode } = useWebApi();
     const model = useCurrent(webApi, 'model', modelId, modelData);
 
     const updateModelProperty = useCallback(
-        (key: string, value: string) => {
+        (key: string, value: string | string[] | UserId | UserId[]) => {
```

Comment:
Let's make this more robust.

```suggestion
        <K extends keyof Model>(key: K, value: Model[K]) => {
```